js/actions/appActions.js

```diff
@@ -959,6 +959,32 @@ const appActions = {
   },
 
   /**
+   * Add records sent with sync lib's SEND_SYNC_RECORDS to the appState
+   * records pending upload. After we download records via the sync lib
+   * we run confirmSyncRecords.
+   * @param {Object} records: Array.<object>
```

Comment:
nit: ':' at end of records

i think this should be called addPendingSyncRecords since i've never seen pend used as a verb